tests/db_migrator_input/config_db/feature-input.json

```diff
@@ -0,0 +1,13 @@
+{
+	"CONTAINER_FEATURE|swss": {
+		"auto_restart": "disabled",
+		"high_mem_alert": "disabled"
+	},
+	"CONTAINER_FEATURE|telemetry": {
+		"auto_restart": "enabled",
+		"high_mem_alert": "disabled"
+	},
+	"FEATURE|telemetry": {
+		"status": "enabled"
```

Comment:
Should this be `state`, instead of `status`?

Reply:
This is input file, should be "status" as this is original schema